config/samples/multipleDynakubes.yml

```diff
@@ -0,0 +1,72 @@
+apiVersion: dynatrace.com/v1beta1
+kind: DynaKube
+metadata:
+  name: dynakube-application-monitoring
+  namespace: dynatrace
+spec:
+  # Dynatrace apiUrl including the `/api` path at the end.
+  # For SaaS, set `YOUR_ENVIRONMENT_ID` to your environment ID.
+  # For Managed, change the apiUrl address.
+  # For instructions on how to determine the environment ID and how to configure the apiUrl address, see https://www.dynatrace.com/support/help/reference/dynatrace-concepts/environment-id/.
+  #
+  apiUrl: https://ENVIRONMENTID.live.dynatrace.com/api
+
+  # Name of the secret holding the API and PaaS tokens.
+  # If unset, Dynatrace Operator uses the name of the custom resource.
+  #
+  # tokens: ""
```

Comment:
Could set tokens for both Dynakubes to the same value and explain that it reuses the Secret